suite-common/fiat-services/src/rates.ts

```diff
@@ -63,6 +67,20 @@ export const fetchCurrentFiatRates = ({
                         { timeout: CONNECT_FETCH_TIMEOUT },
                     );
 
+                    if (!success && payload.error === 'No tickers found!') {
```

Comment:
The error here from connect is pretty bad, jsut a sentence, no error code.

So either this, or [this solution ](https://github.com/trezor/trezor-suite/pull/18354/files#r2042205856)

Or we will make up something even better?

Reply:
`this solution` link points to the same thing.

Yes, the error is just a string as it comes directly from Blockbook, so if you really want to ignore other possible errors, this is the way.

Also, this fallback is needed only for current fiat rates and not last week fiat rates or specific timestamp rates? Why not?

By the way, @tomasklim why have we decided to remove fiat rate fallbacks so now it seems we have to readd (some of) them? I believe it had worked that when Connect failed to return fiat rates, we tried request them directly from Blockbook and if it failed as well, we finally tried coingecko.

Reply:
> Also, this fallback is needed only for current fiat rates and not last week fiat rates or specific timestamp rates? Why not?

I would say it is needed, the question is, if we even need fallback for rates which should be on blockbook but for some reason they are not there

Reply:
> By the way, @tomasklim why have we decided to remove fiat rate fallbacks so now it seems we have to readd (some of) them? I believe it had worked that when Connect failed to return fiat rates, we tried request them directly from Blockbook and if it failed as well, we finally tried coingecko.

Until now I though it works like that but it does not. However, is it even needed? When it is not available on blockbook it is probably not available on coingecko at all (for blockbook based networks)

Reply:
Well, I know of 2 coins that are availabeo n coingecko and not available at blockbook.